src/handler.ts

```diff
@@ -18,15 +18,18 @@ export interface LineDecorationText {
  * @returns repo and file part of URI.
  */
 export function getParamsFromUriPath(textDocumentURI: string): Params {
-    // TODO: Support more than just GitHub & Gitlab.
-    // TODO: Safeguard for cases where repo/fileMatch are null.
-    const repoPattern = /(github\.com|gitlab\.com)\/([^\?\#\/]+\/[^\?\#\/]*)/gi
-    const filePattern = /#(.*\.(.*))$/gi
-    const repoMatch = repoPattern.exec(textDocumentURI)
-    const fileMatch = filePattern.exec(textDocumentURI)
+    if (textDocumentURI) {
```

Comment:
Rather than simply checking for thruthiness of `textDocumentURI`, you might want to:
- check for errors in URL parsing (`new URL(textDocumentURI)`)
- check file match
- return a falsy value (eg. `null`, see my comment below) if either fails.